lib/wor/paginate/paginate.rb

```
@@ -0,0 +1,36 @@
+module Wor
+  module Paginate
+    attr_accessor :adapters
+
+    def initialize
+      self.adapters = [Adapters::KaminariAlreadyPaginated.new,
+                       Adapters::WillPaginateAlreadyPaginated.new,
+                       Adapters::Kaminari.new,
+                       Adapters::WillPaginate.new,
+                       Adapters::ActiveModel.new,
+                       Adapters::Iterable.new]
+    end
+
+    def render_paginated(content)
+      render json: paginate(content)
+    end
+
+    def paginate(content)
+      adapter = find_adapter_for_content(content)
+      throw "No pagination adapter for class #{content.class}" unless adapter.present?
```

Can we raise a custom exception here?